lib/manager/containers/LabelEditor.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import { connect } from 'react-redux'
+
+import { createLabel, updateLabel } from '../../manager/actions/labels'
+import LabelEditor from '../components/LabelEditor'
+
+const mapStateToProps = (state, ownProps) => {
+  return {
+    newLabel: state.newLabel,
+    user: state.user
+  }
+}
+
+const mapDispatchToProps = {
+  createLabel, updateLabel
+}
+
+export default connect(
+  mapStateToProps,
+  mapDispatchToProps
+)(LabelEditor)
```

Comment:
In practice with newer components, I believe we've abandoned the separation of components into connected containers and the components, so it's fine to combine this in just the component file.

Reply:
Refactored in a1bc3e9